cross/json-c/Makefile

```diff
@@ -0,0 +1,18 @@
+PKG_NAME = json-c
+PKG_VERS = 0.13.1-20180305
```

Comment:
Please notice another pending PR #3856 introduces an older version: https://github.com/SynoCommunity/spksrc/blob/47bf32e403de4fe37f717b71bf1ce6d8979ac93f/cross/json-c/Makefile

Reply:
The file in the PR #3856 looks identical to mine (0.13.1) vs the other one you refer to uses an older version (0.12).  I guess I could merge the difference in my PR (e.g. use my newer version + added args from the other)?

Reply:
It **is** identical to your version as I made a copy for this commit 747e1c71bcfd0808de640f9ba42bd43e0aade947 (I could have made a cherry pick, but in your commit were two other files).